### ISSUE TITLE
Posting a status with special characters isn't properly encoded

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NityaNandPandey/Linkedin-SignIn#48]
```
What steps will reproduce the problem?
1. use the postShare method
2. Post the following status "Rêvez vos voyages, et voyagePlus s'en occupe"

The status displayed on Linkedin looks like:
R�vez vos voyages, et voyagePlus s?en occupe

Any solution on how to encode those special characters?

```

Original issue reported on code.google.com by `michaelg...@hotmail.com` on 25 Feb 2011 at 9:05

**Comment by GoogleCodeExporter** [comment]
```
Thanks for pointing this out. I will fix that in the next build.
```

Original comment by `nabeelmukhtar` on 25 Feb 2011 at 9:51
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Thank you, I'll be waiting for it ;)
```

Original comment by `michaelg...@hotmail.com` on 25 Feb 2011 at 2:08

**Comment by GoogleCodeExporter** [comment]
```
This issue is fixed in build 1.0.367.
```

Original comment by `nabeelmukhtar` on 5 Apr 2011 at 12:58
- Changed state: **Fixed**
